src/loaders/png/tvgPngLoader.cpp

```diff
@@ -213,5 +213,7 @@ void PngLoader::run(unsigned tid)
 
     lodepng_decode(&image, &width, &height, &state, data, size);
 
+    if (state.info_png.color.colortype == LCT_RGBA) colorSpace = SwCanvas::ABGR8888;
```

Comment:
If possible the values must be updated earlier. So it looks open() is a better place to put?

Reply:
I move the code. thanks